lib/internal/modules/esm/translators.js

```diff
@@ -199,14 +200,18 @@ function createCJSModuleWrap(url, source, isMain, loadCJS = loadCJSModule) {
 
   const { exportNames, module } = cjsPreparseModuleExports(filename, source);
   cjsCache.set(url, module);
-  const namesWithDefault = exportNames.has('default') ?
-    [...exportNames] : ['default', ...exportNames];
+
+  const wrapperNames = [...exportNames];
+  if (!exportNames.has('default')) {
+    ArrayPrototypePush(wrapperNames, 'default');
+  }
+  ArrayPrototypePush(wrapperNames, 'module.exports');
```

Comment:
Do we care about the order? If so, we should add a comment. If not, can this be simplified to: 
```suggestion
  const wrapperNames = [...exportNames, 'module.exports'];
  if (!exportNames.has('default')) {
    ArrayPrototypePush(wrapperNames, 'default');
  }
```

Reply:
Order here doesn't matter since it gets sorted on the other side anyway. Added.